### DISCUSSION TITLE
VRM1.0対応SDKについて

### DISCUSSION THREAD

**Discussion by elvneko** [pixiv/vroid-sdk-developers#50] [General]
日頃大変お世話になっております。

Vroid SDKを使わせていただけてるおかげでユーザー様に大変ご好評いただけております。

8月16日の「VRMコンソーシアム: 2022セミナー」にて、
`ピクシブの「VRoid Hub」は、9月上旬に対応予定です。VRM 1.0は、このまま順調に進めば、2022年9月中に正式版として発行される予定です。`という内容がありました。

https://www.moguravr.com/vrm-consortium-2022-seminar-report/

これに伴いVRM1.0に対応したSDKの配布予定はあるでしょうか。

当方開発しておりますアプリのSteam版はすぐにアップデートを配信できるから良いのですが、モバイル向けストアでは配信までに審査を挟む必要があり早めに行動しておかないとタイムラグが生じてしまいます。
そのため「VRMが読み込めない」とユーザー様から問い合わせが来ることを危惧しております。

お忙しいところ恐縮ですが、
ご回答いただけると幸いです。

**Comment by elvneko** [comment]


**Reply by nyamadan** [reply]
VRoid SDKのVRM1対応の計画はございますが、配布日程の目途は立っておりません。

また、配布中のSDKについてはVRM1のモデルは利用できない仕様となっております。
そのため、VRM1モデルをアップロードしたがVRoid SDKでは利用できないという状況が発生する可能性はございます。

情報公開が遅れてしまい、大変申し訳ございませんでした。

**Comment by RearAlice** [comment]
自分が聞きたいことの関連ディスカッションだったので失礼します。

上記スレッドで「VRoid SDKのVRM1対応の計画はございますが、配布日程の目途は立ってない」とのことですが、
アプリケーションの配布ページ等に現状VRM1モデルは利用できない趣旨は書いておいたほうがいいんでしょうか？

**Reply by ytomi-pixiv** [reply]
ご指摘ありがとうございます。お手数おかけして申し訳ありません。

もし可能であれば、VRoid Hub以外の配布ページ等でも記載いただけますと幸いです。

VRoid Hubとしては、モデル利用者に向けてモデル詳細ページ等でVRM1.0がSDK経由で利用できない旨をアナウンスしております。重ねて、今回の指摘を受け、VRoid Hubにおける各アプリケーションの詳細ページに、VRoid HubにアップロードしたVRM1.0モデルは利用できない旨を追記いたしました。

VRM1.0対応の計画はございますので、今しばらくお待ちいただけますと幸いです。
不明点等ありましたら、お知らせください。引き続きよろしくお願いいたします。

**Reply by elvneko** [reply]
VRoid Hubでのアナウンスのおかげか、自分のアプリユーザーからはvrm1.0に関する問い合わせはあまり来ておりません。

Vroidプロジェクトのおかげで現在フルタイムの個人開発者として活動できているので大変感謝しております。VRM1.0がSDKでサポートされたら連携アプリ・ゲームを更に追加で開発していき、微力ながらもVroidの更なる盛り上がりに貢献できればと思います。

気長に待っていますのでチームのみなさんのお身体に無理のなさらぬよう開発いただければ幸いです。
